### ISSUE TITLE
Unable to install OwnCloud - Please check your project settings and ensure that a valid product has been built.

### ISSUE THREAD

**Issue by ahmedsaeed548** [owncloud/ios-app#1232]
Hello,

I have configured ownCloud, and followed every step except the MIT License one. 

My app builds successfully, but when It is copying files and launching, a box pops up with the error : 

It appears while copying pkginfo.

**_"Unable to install OwnCloud - Please check your project settings and ensure that a valid product has been built."_**

and on Simulator error: 

**_Simulator device returned an error for the requested operation._**

I have tried each and everything but all in vain. Deleted Drives data, reinstalled SwiftLint etc. Everything is configured correctly. 

Do I need to add MIT License to make it work? because I guess that's for later purpose incase if I want to add something in code. 

NOTE: Project builds successfully. (No error in Xcode) 

```
Details

Unable to install "ownCloud"
Domain: com.apple.dt.MobileDeviceErrorDomain
Code: -402653081
Recovery Suggestion: Please check your project settings and ensure that a valid product has been built.
User Info: {
    DVTErrorCreationDateKey = "2023-06-26 11:29:22 +0000";
    IDERunOperationFailingWorker = IDEInstalliPhoneLauncher;
}
--
There was an internal API error.
Domain: com.apple.dt.MobileDeviceErrorDomain
Code: -402653081
User Info: {
    DVTRadarComponentKey = 261622;
    MobileDeviceErrorCode = "(0xE8000067)";
    "com.apple.dtdevicekit.stacktrace" = (
	0   DTDeviceKitBase                     0x000000010bba4b1c DTDKCreateNSErrorFromAMDErrorCode + 300
	1   DTDeviceKitBase                     0x000000010bbd9000 __90-[DTDKMobileDeviceToken installApplicationBundleAtPath:withOptions:andError:withCallback:]_block_invoke + 136
	2   DVTFoundation                       0x0000000102107480 DVTInvokeWithStrongOwnership + 76
	3   DTDeviceKitBase                     0x000000010bbd8d7c -[DTDKMobileDeviceToken installApplicationBundleAtPath:withOptions:andError:withCallback:] + 1208
	4   IDEiOSSupportCore                   0x000000010ba8fc00 __118-[DVTiOSDevice(DVTiPhoneApplicationInstallation) processAppInstallSet:appUninstallSet:installOptions:completionBlock:]_block_invoke.147 + 2340
	5   DVTFoundation                       0x0000000102211de0 __DVT_CALLING_CLIENT_BLOCK__ + 16
	6   DVTFoundation                       0x000000010221284c __DVTDispatchAsync_block_invoke + 152
	7   libdispatch.dylib                   0x00000001aba2c9dc _dispatch_call_block_and_release + 32
	8   libdispatch.dylib                   0x00000001aba2e504 _dispatch_client_callout + 20
	9   libdispatch.dylib                   0x00000001aba35bbc _dispatch_lane_serial_drain + 672
	10  libdispatch.dylib                   0x00000001aba36708 _dispatch_lane_invoke + 384
	11  libdispatch.dylib                   0x00000001aba4134c _dispatch_workloop_worker_thread + 652
	12  libsystem_pthread.dylib             0x00000001abbb2100 _pthread_wqthread + 288
	13  libsystem_pthread.dylib             0x00000001abbb0e20 start_wqthread + 8
);
}
--

Analytics Event: com.apple.dt.IDERunOperationWorkerFinished : {
    "device_model" = "iPhone10,6";
    "device_osBuild" = "16.5 (20F66)";
    "device_platform" = "com.apple.platform.iphoneos";
    "launchSession_schemeCommand" = Run;
    "launchSession_state" = 1;
    "launchSession_targetArch" = arm64;
    "operation_duration_ms" = 9770;
    "operation_errorCode" = "-402653081";
    "operation_errorDomain" = "com.apple.dt.MobileDeviceErrorDomain";
    "operation_errorWorker" = IDEInstalliPhoneLauncher;
    "operation_name" = IDEiPhoneRunOperationWorkerGroup;
    "param_consoleMode" = 0;
    "param_debugger_attachToExtensions" = 0;
    "param_debugger_attachToXPC" = 1;
    "param_debugger_type" = 5;
    "param_destination_isProxy" = 0;
    "param_destination_platform" = "com.apple.platform.iphoneos";
    "param_diag_MainThreadChecker_stopOnIssue" = 0;
    "param_diag_MallocStackLogging_enableDuringAttach" = 0;
    "param_diag_MallocStackLogging_enableForXPC" = 1;
    "param_diag_allowLocationSimulation" = 1;
    "param_diag_checker_tpc_enable" = 1;
    "param_diag_gpu_frameCapture_enable" = 0;
    "param_diag_gpu_shaderValidation_enable" = 0;
    "param_diag_gpu_validation_enable" = 0;
    "param_diag_memoryGraphOnResourceException" = 0;
    "param_diag_queueDebugging_enable" = 1;
    "param_diag_runtimeProfile_generate" = 0;
    "param_diag_sanitizer_asan_enable" = 0;
    "param_diag_sanitizer_tsan_enable" = 0;
    "param_diag_sanitizer_tsan_stopOnIssue" = 0;
    "param_diag_sanitizer_ubsan_stopOnIssue" = 0;
    "param_diag_showNonLocalizedStrings" = 0;
    "param_diag_viewDebugging_enabled" = 1;
    "param_diag_viewDebugging_insertDylibOnLaunch" = 1;
    "param_install_style" = 0;
    "param_launcher_UID" = 2;
    "param_launcher_allowDeviceSensorReplayData" = 0;
    "param_launcher_kind" = 0;
    "param_launcher_style" = 0;
    "param_launcher_substyle" = 0;
    "param_runnable_appExtensionHostRunMode" = 0;
    "param_runnable_productType" = "com.apple.product-type.application";
    "param_testing_launchedForTesting" = 0;
    "param_testing_suppressSimulatorApp" = 0;
    "param_testing_usingCLI" = 0;
    "sdk_canonicalName" = "iphoneos16.4";
    "sdk_osVersion" = "16.4";
    "sdk_variant" = iphoneos;
}
--


System Information

macOS Version 13.2.1 (Build 22D68)
Xcode 14.3.1 (21815) (Build 14E300c)
Timestamp: 2023-06-26T16:29:22+05:00
```

<img width="821" alt="Screenshot 2023-06-26 at 4 34 17 PM" src="https://github.com/owncloud/ios-app/assets/57411594/4637aac6-54ce-40c1-bcc5-308368695077">

**Comment by michaelstingl** [comment]
Cant't reproduce. It installs fine in Simulator following the steps:

![CleanShot 2023-06-26 at 14 23 02](https://github.com/owncloud/ios-app/assets/214010/6cca98df-ff30-470e-b642-6d5265daadfe)

```
macOS 13.4.1 (22F82)
Xcode 14.3.1 (14E300c)
```

Did you perform the `submodule` steps?
- https://github.com/owncloud/ios-app/blob/master/SETUP.md#2-add-the-owncloud-ios-sdk

**Comment by ahmedsaeed548** [comment]
Yes, I followed all the steps till 4. I didn't follow 5 & 6 yet.. I will follow it once I need to contribute which I may be doing so but not sure. 
iOS SDK is installed properly. 
There is no error in the project while building. I am stuck on this for 2 days.

**Comment by ahmedsaeed548** [comment]
Issue Fixed.

I click on the details to see the error message and there I found error related to Plugin placeholders. 

I figured out that it does not accept the bundle identifies having "-" in it.

for example: com.name.owncloud.ios-app-share-intents (not accepted)
accepted identifer: com.name.owncloud.ios.app.share.intents

Link to resolve issue: 
https://stackoverflow.com/questions/47344160/failed-to-set-plugin-placeholders-message
